# request
string target
---
#response
bool destinationReached
